diff -durN Zope-2.7.0-b2.orig/configure Zope-2.7.0-b2/configure
--- Zope-2.7.0-b2.orig/configure	Fri Aug 22 16:07:55 2003
+++ Zope-2.7.0-b2/configure	Wed Nov 12 17:42:58 2003
@@ -19,7 +19,7 @@
 # sys.version) below in "best" to "worst" order, not including the
 # target version.  Up to six acceptable python versions are allowed.
 # Do not include the target version number in this list!
-ACCEPTABLE="2.2.2 2.3 2.3b1 2.3a1 2.3a0"
+ACCEPTABLE="2.2.2 2.3 2.3b1 2.3a1 2.3a0 2.3.2"
 
 # provide the executable names for all the acceptable versions
 # (and the target version) below
@@ -113,7 +113,8 @@
         out ""
         out "  The optimimum Python version ($TARGET) was found at $FOUND."
     elif [ -z "$FOUND1" ] && [ -z "$FOUND2" ] && [ -z "$FOUND3" ] &&
-         [ -z "$FOUND4" ] && [ -z "$FOUND5" ] && [ -z "$FOUND6" ] ; then
+         [ -z "$FOUND4" ] && [ -z "$FOUND5" ] && [ -z "$FOUND6" ] &&
+	 [ -z "$FOUND7" ] ; then
         out ""
         out "  No suitable Python version found.  You should install"
         out "  Python version $TARGET before continuing.  Versions"
@@ -138,6 +139,9 @@
         elif [ -n "$FOUND6" ]; then
             FOUND=$FOUND6
             FOUNDVERSION=$FOUNDVERSION6
+        elif [ -n "$FOUND7" ]; then
+            FOUND=$FOUND7
+            FOUNDVERSION=$FOUNDVERSION7
         fi
         out ""
         out "  !! WARNING !! "
