--- Zope-2.6.2b5-src.orig/lib/python/Products/StandardCacheManagers/AcceleratedHTTPCacheManager.py 2002-08-14 18:25:12.000000000 -0400
+++ Zope-2.6.2b5-src/lib/python/Products/StandardCacheManagers/AcceleratedHTTPCacheManager.py 2003-02-27 16:26:02.000000000 -0500
@@ -45,7 +45,7 @@
         phys_path = ob.getPhysicalPath()
         if self.hit_counts.has_key(phys_path):
             del self.hit_counts[phys_path]
-        ob_path = quote('/'.join(phys_path))
+        ob_path = '/'+ob.absolute_url(relative=1)
         results = []
         for url in self.notify_urls:
             if not url:
@@ -57,6 +57,8 @@
                 u = 'http://' + url
             (scheme, host, path, params, query, fragment
              ) = urlparse.urlparse(u)
+            if path[:8].lower() == '/http://':
+                path = path[1:]
             if path[-1:] == '/':
                 p = path[:-1] + ob_path
             else:
@@ -67,6 +69,12 @@
             errcode, errmsg, headers = h.getreply()
             h.getfile().read()  # Mandatory for httplib?
             results.append('%s %s' % (errcode, errmsg))
+            if p[-11:]=='/index_html':
+                h.putrequest('PURGE',p[:-10])
+                h.endheaders()
+                errcode, errmsg, headers = h.getreply()
+                h.getfile().read()  # Mandatory for httplib?
+                results.append('%s %s' % (errcode, errmsg))
         return 'Server response(s): ' + ';'.join(results)

     def ZCache_get(self, ob, view_name, keywords, mtime_func, default):
