--- Zope-2.7.3-0/configure.orig	2004-12-23 11:57:58.812776192 +0000
+++ Zope-2.7.3-0/configure	2004-12-23 11:58:14.666366080 +0000
@@ -12,17 +12,17 @@
 
 # Place the optimal target version number for Zope (as returned by sys.version)
 # below
-TARGET="2.3.4"
+TARGET="2.4"
 
 # Order a list of "acceptable" python version numbers (as returned by
 # sys.version) below in "best" to "worst" order, not including the
 # target version.  Up to six acceptable python versions are allowed.
 # Do not include the target version number in this list!
-ACCEPTABLE="2.3.3"
+ACCEPTABLE="2.3.4 2.3.3"
 
 # provide the executable names for all the acceptable versions
 # (and the target version) below
-EXENAMES="python python2 python2.3"
+EXENAMES="python python2 python2.4 python2.3"
 
 #####################################################################
 #                    END EDITABLE PARAMETERS                        #
