diff -urN Zope-2.7.5-final.org/configure Zope-2.7.5-final/configure
--- Zope-2.7.5-final.org/configure	2005-03-18 14:21:34.000000000 +0100
+++ Zope-2.7.5-final/configure	2005-03-18 22:37:32.568418896 +0100
@@ -12,7 +12,7 @@
 
 # Place the optimal target version number for Zope (as returned by sys.version)
 # below
-TARGET="2.3.5"
+TARGET="2.4.1"
 
 # Order a list of "acceptable" python version numbers (as returned by
 # sys.version) below in "best" to "worst" order, not including the
@@ -22,7 +22,7 @@
 
 # provide the executable names for all the acceptable versions
 # (and the target version) below
-EXENAMES="python python2 python2.3"
+EXENAMES="python python2 python2.3 python2.4"
 
 #####################################################################
 #                    END EDITABLE PARAMETERS                        #
