diff -urN Zope-2.7.0/lib/python/zdaemon/zdctl.py Zope-2.7.0_chng/lib/python/zdaemon/zdctl.py
--- Zope-2.7.0/lib/python/zdaemon/zdctl.py	2003-10-24 23:32:59.000000000 +0200
+++ Zope-2.7.0_chng/lib/python/zdaemon/zdctl.py	2004-03-23 13:08:44.808358240 +0100
@@ -208,9 +208,9 @@
             args += self._get_override("-z", "directory")
             args.extend(self.options.program)
             if self.options.daemon:
-                flag = os.P_WAIT
-            else:
                 flag = os.P_NOWAIT
+            else:
+                flag = os.P_WAIT
             os.spawnvp(flag, args[0], args)
         elif not self.zd_pid:
             self.send_action("start")
